remotemonitoring/single-vm/setup.sh

```diff
@@ -73,9 +74,138 @@ cd ${APP_PATH}
 
 # Docker compose file
 
-# Note: the "APP_RUNTIME" var needs to be defined before getting here
-DOCKERCOMPOSE_SOURCE="${REPOSITORY}/docker-compose.${APP_RUNTIME}.yml"
-wget $DOCKERCOMPOSE_SOURCE -O ${DOCKERCOMPOSE}
+touch ${DOCKERCOMPOSE} && chmod 550 ${DOCKERCOMPOSE}
+echo "version: '2'"                                                                         >> ${DOCKERCOMPOSE}
+echo ""                                                                                     >> ${DOCKERCOMPOSE}
+echo "services:"                                                                            >> ${DOCKERCOMPOSE}
+echo "  reverseproxy:"                                                                      >> ${DOCKERCOMPOSE}
+echo "    image: azureiotpcs/remote-monitoring-nginx:${APP_RELEASE_VERSION}"                >> ${DOCKERCOMPOSE}
+echo "    ports:"                                                                           >> ${DOCKERCOMPOSE}
+echo "      - \"80:80\""                                                                    >> ${DOCKERCOMPOSE}
```

Comment:
btw, we should remove port 80 before public preview

Reply:
I can remove it now if it is not needed, i wasn't sure.